packages/terser/src/terser_minified.bzl

```diff
@@ -138,6 +138,11 @@ def _terser(ctx):
         else:
             fail("When sourcemap is True, there should only be one or none input sourcemaps")
 
+        if len(directory_srcs) == 0:
+            source_map_opts.append("url='%s.js.map'" % ctx.label.name)
+        else:
+            source_map_opts.append("url='<OUTPUT_MAP_FILE>'")
```

Comment:
a comment that this special key is replaced with the actual filename in the .js file would be good here